lib/ansible/modules/windows/win_regedit.ps1

```diff
@@ -21,313 +22,317 @@
 
 $ErrorActionPreference = "Stop"
 
-$params = Parse-Args $args -supports_check_mode $true
+$params = Parse-Args -arguments $args -supports_check_mode $true
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
-$diff_support = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -default $false
+$diff_mode = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -default $false
```

Comment:
I just noticed that `$diff_support` was renamed to `$diff_mode`. However that name was chosen on purpose. You can enable diff-support in both run-mode and check-mode, so it's not a mode in itself.